message_center_compassion/model/gmc_message.py

```diff
@@ -0,0 +1,433 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2014 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emanuel Cino <ecino@compassion.ch>
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+from openerp import api, models, fields, _
+from openerp.exceptions import Warning
+from openerp.tools import DEFAULT_SERVER_DATE_FORMAT as DF
+from openerp.tools.config import config
+
+from random import randint
+from datetime import datetime
+
+import requests
+import logging
+import traceback
+
+logger = logging.getLogger(__name__)
+
+
+class gmc_message_pool_process(models.TransientModel):
+    _name = 'gmc.message.pool.process'
+
+    @api.multi
+    def process_messages(self):
+        active_ids = self.env.context.get('active_ids')
```

Comment:
Are we sure that `active_ids` is in the context? Wouldn't be safer to use get like that: `self.env.context.get('active_ids',[])` ?